
+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/bcmdrivers/opensource/include/bcm963xx/xrdp_wfd_inline.h
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/bcmdrivers/opensource/include/bcm963xx/xrdp_wfd_inline.h

@@ -245,7 +245,10 @@
             /* Check if device was initialized */
             if (unlikely(!wfd_p->wl_if_dev[ifid]))
             {
-                printk("%s wifi_net_devices[%d] returned NULL\n", __FUNCTION__, ifid);
+                if (printk_ratelimit()) 
+                {
+                    printk("%s wifi_net_devices[%d] returned NULL\n", __FUNCTION__, ifid);
+                }
                 continue;
             }
 
